src/cairo_run.rs

```diff
@@ -23,6 +26,7 @@ pub fn cairo_run_py<'a>(
     print_output: bool,
     trace_file: Option<&str>,
     memory_file: Option<&str>,
+    hint_locals: Option<HashMap<String, PyObject>>,
```

Comment:
In which cases would this be `None`? It looks like we could simply assume one to be present, even if an empty one, to have code be a little bit simpler.

Reply:
Option is used so that hint_locals doesn't need to be specified when calling the function from python

Reply:
Nice. I didn't know PyO3 handled that too.